exotic.py

```diff
@@ -1717,6 +1709,35 @@ def parse_args():
             timesListed = []
             airMassList = []
 
+            plate_opt = str(input("\nDisclaimer: Your .FITS file will be publicly viewable on nova.astrometry.net. "
+                                  "Would you like to upload it for a plate solution? (y/n) "))
+
+            while plate_opt.lower() != 'y' and plate_opt.lower() != 'n':
+                plate_opt = str(input("Not a valid command. Would you like to upload your .FITS file? (y/n) "))
+            if plate_opt.lower() == 'y':
+                print("\nGetting the plate solution for your first .FITS file. Please wait.")
+                done = False
+                t = threading.Thread(target=animate, daemon=True)
+                t.start()
+
+                # Gets the first .fits file from the directory
+                first_fits = inputfiles[0]
+
+                # Plate solves the .fits file and returns status
+                WCS_fits_file = plate_solution(first_fits, saveDirectory)
+
+                # Get WCS for the first .fits file to track x,y coordinates
+                # WCS_fits_data = WCS_fits_file.getdata(WCS_fits_file)
+                done = True
+
+            #Doublechecking to see if comparison star != variable star
+            print("Verifying selected comparison star is not variable. Please wait.")
```

Comment:
I don't think this is the right place for the comp star checker as this is not in the for loop that loops over the multiple comparison stars. Shouldn't this be after line that has `for compCounter in range(0, len(compStarList)):` instead? That way if the comparison star chosen is variable, the for loop skips over it. Or perhaps

Reply:
I moved this to be right after the part that determines the optimal apertur for the comparison star as suggested since by then we'll have a list of comparison stars to go through.

Reply:
Please see updated code